Taller152/src/paquetetres/Principal.java

```diff
@@ -5,10 +5,113 @@
  */
 package paquetetres;
 
+import java.util.Scanner;
+
 /**
  *
  * @author reroes
  */
 public class Principal {
     
-}
+    public static void main(String[] args){
+        Scanner entrada = new Scanner(System.in);
+        String[] libros = {"El camino a un mejor programador", 
+            "Scrum & eXtreme Programming", 
+            "Lógica de programación", 
+            "Metodología programación orientada a objetos", 
+            "Java Como Programar", 
+            "Python 3 al descubierto"};
+        
+        String[] autores = {"Anónimo", 
+            "Eugenia Bahit", 
+            "Omar Iván Trejos Buriticá", 
+            "López Román Leobardo", 
+            "Dietel, P. y Dietel, H", 
+            "Fernández Arturo"};
+        
+        int[] clave1 = {2, 1, 0, 2, 0, 1};
+        
+        int[] clave2 = {1, 1, 1, 2, 2, 0};
+        
+        int[][] identificativoLibro = {
+            {8761, 12334, 34567},
+            {65431, 43211, 7890},
+            {123890, 12344, 2345}
+        };
+        
+        int opcion;
+        String cadena = "";
+        
+        System.out.printf("Libros a prestar:\n");
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.